### ISSUE TITLE
BlenderBIM - Error when using the `extend` function from BIM Tool

### ISSUE THREAD

**Issue by brunoperdigao** [IfcOpenShell/IfcOpenShell#3431]
To reproduce, add a beam and press `shift+e`. It throws this error:

```
Python: Traceback (most recent call last):
  File "C:\Users\Bruno\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\model\workspace.py", line 384, in invoke
    return self.execute(context)
  File "C:\Users\Bruno\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\tool\ifc.py", line 136, in execute
    IfcStore.execute_ifc_operator(self, context)
  File "C:\Users\Bruno\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\ifc.py", line 421, in execute_ifc_operator
    result = getattr(operator, "_execute")(context)
  File "C:\Users\Bruno\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\model\workspace.py", line 376, in _execute
    getattr(self, f"hotkey_{self.hotkey}")()
  File "C:\Users\Bruno\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\model\workspace.py", line 429, in hotkey_S_E
    (RailingData.is_loaded or not RailingData.load())
  File "C:\Users\Bruno\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\model\data.py", line 369, in load
    cls.data["general_params"] = cls.general_params()
  File "C:\Users\Bruno\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\model\data.py", line 379, in general_params
    railing_data = cls.data["parameters"]["data_dict"]
TypeError: 'NoneType' object is not subscriptable

```

**Comment by Moult** [comment]
The fix is simple but assigning to @Andrej730 since he has been in the middle of refactoring this so he'll be aware if there are other similar problems elsewhere.

@Andrej730 also can you make sure key and function names always match? e.g. `cls.data["parameters"] = cls.pset_data()` should either be parameters or pset_data.

**Comment by Andrej730** [comment]
Fixed in 96d812eab
